site/src/pages/RoadmapPage/Year.scss

```diff
@@ -41,14 +41,21 @@
   margin-right: 10px;
 }
 
-.year-accordion-content {
+.year-accordion-content,
+.year-accordion-content-3-quarter {
```

Comment:
if we use the base & subclass, then this style rule would remain at one class since the base class would apply to both situations.